test/random.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>QUnit Randomization Test Suite</title>
+    <link rel="stylesheet" href="../dist/qunit.css">
+    <script src="../dist/qunit.js"></script>
+    <script>
+      QUnit.config.seed = "7x9";
```

Comment:
Could you move this into `random.js`? It's a little weird to see separation from the code assuming it. After that, I think we're good to go. Thanks @trentmwillis!